### PR TITLE
Resolved fetchPriority prop warning

### PR DESCRIPTION
**Pull request by Faith-Nchifor**
|                                    Web Dev Path                                    |
| :--------------------------------------------------------------------------------: |
| [**227**](https://github.com/MarianaSouza/web-dev-path/issues/issueNumber) |

#### Have you updated the CHANGELOG.md file? If not, please do it.
Done

#### What is this change?
The  fetchPriority prop warning in next/image was resolved by upgrading NextJs to 14.2.20   

#### Were there any complications while making this change?
No

#### How to replicate the issue?
I ran `yarn run dev` and waited for the homepage to load

#### If necessary, please describe how to test the new feature or fix.
Run `yarn run dev` and check the console to see if the warnings will appear.
Capture

#### When should this be merged?
After reviews

### CODE REVIEW

**Review by Satoshi-Sh — APPROVED** [review]
It looks good to me. Thanks for taking on the issue.

Upgrading Next also seems fix those errors. They are gone on the deploy preview. We can close #213 once this PR gets merged.

image-errors

**Review by cherylli — COMMENTED** [review]
I have also checked that the fetchPriority warning is gone now (local)

File: CHANGELOG.md [hunk 1]
```diff
@@ -139,3 +139,4 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed contact form message position
 - Fixed blog images not loading due to dev.to changing image host (sub)domain
 - Fixed empty blog category by using per_page parameter
+- Resolved `fetchPriority` prop warning in next/image
```
[the remark this answers is not in the document]
Comment (mariana-caldas) [reply CHANGELOG.md:142]: Good thinking, @cherylli !

**Review by mariana-caldas — APPROVED** [review]
Thanks so much, @Faith-Nchifor ! Welcome to the club! 🥇

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Faith-Nchifor) [reply CHANGELOG.md:142]: Should I update the changelog file @mariana-caldas ?

**Comment by Faith-Nchifor** [comment]
> Thanks so much, @Faith-Nchifor ! Welcome to the club! 🥇

Thank you :)

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mariana-caldas) [reply CHANGELOG.md:142]: Hey @Faith-Nchifor ! Yes, please. It's a good practice to go through all the team members' feedback and then mark each conversation as solved before you merge the PR (we merge it the simplest way with a merge commit, btw) Thanks so much!

**Review by cherylli — APPROVED** [review]

**Pull request merged**